sopel/modules/currency.py

```diff
@@ -215,28 +216,25 @@ def update_rates(bot):
     rates_crypto = response.json()
 
     # Update fiat rates
-    if bot.config.currency.fixer_io_key is not None:
+    settings = bot.settings.currency
```

Comment:
I have mixed feelings about making such aliases. To someone unfamiliar with the structure of a Sopel instance, reading core plugin code to learn from existing examples, this is easy to miss (and then later wonder where the hell `settings` comes from).

In this particular case, converting to use this alias also pollutes the diff 😝

Reply:
Yeah, I'll change that to something easier to read, even for me this is not optimal.